mac-requirements.txt

```diff
@@ -0,0 +1,2 @@
+bs4~=0.0.1
```

Comment:
ahh so i guess i was'nt clear you can leave bs4 in the normal `requirements.txt` and just have `appscript` everybody needs to run `pip instal -r requirements.txt` mac users need to run an additional `pip install -r mac-requirements.txt` . This is important as we have more dependencies theres no point of them being copied twice.